scss/style.scss

```diff
@@ -0,0 +1,585 @@
+    top: 0;
+    left: 15px;
+    width: 25px;
+
+    a {
+      color: #000;
+    }
+
+    .font-icon {
+      background-color: #fff;
+      width: 20px;
+      height: 20px;
+      border-radius: 50%;
+      padding: 3px;
+      margin: 5px 0;
+      cursor: pointer;
+    }
+  }
+}
+
+.thumbs-buttons {
+  @include screen-size('mobile') {
+    display: none;
+  }
+
+  @include screen-size('not-mobile') {
+    color: #fff;
+    font-size: 22px;
+    position: absolute;
+    top: 280px;
+    width: 20px;
+    left: 340px;
+
+    .font-icon {
+      margin: 4px 0;
+      cursor: pointer;
+    }
+  }
+}
+
+.song-data {
+  margin: 20px 0;
+
+  p {
+    margin: 5px;
+  }
+
+  .author {
+    color: #fff;
+    text-align: center;
+  }
+
+  .title {
+    color: #fff;
+    text-align: center;
+    font-size: 20px;
+  }
+
+  .info {
+    font-size: 13px;
+    color: $yellow;
+
+    @include screen-size('mobile') {
+      display: none;
+    }
+  }
+
+  @include screen-size('not-mobile') {
+    position: absolute;
+    top: 70%;
+    left: 15px;
+
+    .author, .title {
+      text-align: left;
+    }
+  }
+}
+
+.button-wrapper {
+  width: 138px;
+  margin: 10px auto 0 auto;
+
+  @include screen-size('not-mobile') {
+    position: absolute;
+    top: -91px;
+    left: 200px;
+
+    .font-icon {
+      margin-right: -10px;
+    }
+  }
+
+  .font-icon {
+    cursor: pointer;
+    background-color: $yellow;
+  }
+
+  .stop-icon {
+    border-radius: 14px;
+    padding: 6px;
+  }
+
+  .play-icon {
+    border-radius: 28px;
+    padding: 14px;
+    font-size: 24px;
+    padding-left: 19px;
+  }
+
+  .forward-icon {
+    border-radius: 16px;
+    padding: 6px;
+    padding-left: 9px;
+  }
+}
+
+.recently-played {
+  .song-wrapper {
+    margin: 20px 10px;
+  }
+
+  p {
+    margin: 0 0 0 70px;
+  }
+
+  img {
+    margin-top: 5px;
+    float: left;
+  }
+
+  .song-author {
+    font-size: 15px;
+  }
+
+  .song-duration {
+    color: $grey;
+    font-weight: 700;
+  }
+}
+
+.concerts {
+  .update-date {
+    color: $grey;
+    font-size: 13px;
+  }
+
+  .concert-details {
+    font-size: 14px;
+  }
+
+  .more-button {
+    color: $yellow;
+    margin: 15px 0 0 130px;
+    width: 70px;
+
+    .more-icon {
+      border-radius: 15px;
+      border: 1px solid $yellow;
+      padding-left: 1px;
+    }
+  }
+}
+
+.contests {
+  .contest-date, .contest-details {
+    color: $grey;
+    font-size: 13px;
+  }
+}
+
+.perks {
+  .perk-date, .perk-details {
+    color: $grey;
+    font-size: 13px;
+  }
+}
+
+.links {
+  p {
+    margin: 0;
+    display: inline-block;
+  }
+
+  .more-icon {
+    float: right;
+  }
+}
+
+.stations {
+  p {
+    margin: 0;
+    display: inline-block;
+    width: 80%;
+  }
+
+  .radio-location, .radio-details {
+    color: $grey;
+    font-size: 11px;
+  }
+
+  a {
+    float: right;
+
+    .more-icon {
+      bottom: 20px;
+      position: relative;
+    }
+  }
+}
+
+@include screen-size('mobile') {
+  .currently-played {
+    height: 545px;
```

Comment:
No fixed height, mobile screens are different in height. Find the way to always show the 100% height background and first screen on mobile